golos.publication/golos.publication.cpp

```diff
@@ -467,6 +469,18 @@ void publication::set_vote(account_name voter, account_name author, string perml
         item.curatorsw = (fixp_t(sumcuratorsw_delta * curatorsw_factor)).data();
         item.rshares = rshares.data();
     });
+
+    tables::forumprops_singleton props_single(_self, _self);
+    auto props = props_single.get_or_default();
+
+    if (props.socialon) {
+        action(
+            permission_level{_self, N(active)},
+            _self,
```

Comment:
`_self` ?
To which contract do you send `changereput` event?

Reply:
@afalaleev To social contract. I had tried it but _self (with require_auth self) has not worked - contract don't receives action